.github/workflows/podvm_smoketest.yaml

```diff
@@ -0,0 +1,157 @@
+name: Run smoke test on PodVM
+
+on:
+  pull_request:
+
+defaults:
+  run:
+    working-directory: src/cloud-api-adaptor/podvm-mkosi
+
+jobs:
+  smoke-test:
+    runs-on: 'ubuntu-22.04'
```

Comment:
Is it worth adding this comment:
> nb: the wf is running on 22.04 b/c libvirt struggles with OVMF_CODE_4M firmware that is default on ub 24.04.

as a workflow comment, so that someone (probably me) doesn't come and mistakenly try and update the runner in the future?